crates/stc_ts_file_analyzer/tests/pass/types/keyof/3.swc-stderr

```diff
@@ -19,7 +19,7 @@ Error:
     `----
 
 Error: 
-  > ("a" | "c" | string | number | symbol)
```

Comment:
Same here

Reply:
in tsc ([link](https://www.typescriptlang.org/ko/play?#code/PTACGcBcCcEsGNIC4AEMCuBTAUCFoATTeAGwENozJYB7AO1Qx21LPHBQEEUBvbFASjKoocOgHN+g+KjroAtgCNM0bAF9s2TAA8ADjWiQ0AT12YUAaUzGOAXhQBrazQBmKABTcAPkLrGAlJpErNDmAG4UjtbgqFY2QcTkoSgkmEYuNDQiMLASKD5ySir5KODGSjQkmhk0KPZO8Q12KDVAA)), `foo`, `keys` are both inferred as (string | number | symbol). 

I don't understand why it should be inferred as ("a" | "c" | string | number | symbol)

Is there something I'm missing or need to know?

Reply:
Those strings are important for a language server

Reply:
The information shown on hover is not a complete information.